kuma/wiki/jinja2/wiki/list/documents.html

```diff
@@ -12,6 +12,7 @@
   {% set title = _('All documents') %}
 {% endif %}
 {% block title %}{{ page_title(title) }}{% endblock %}
+{% block robots_value %}noindex, nofollow{% endblock %}
```

Comment:
This page has been blocked in robots.txt since 2016 in https://github.com/mozilla/kuma/commit/7ef8e07b2d31e2179f81206e2c26faebf8dd1570, as part of the downtime mitigation for [bug 1199166](https://bugzilla.mozilla.org/show_bug.cgi?id=1199166).  I'm not sure I agree with that, but this makes it consistent.